<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformationDockContents</class>
 <widget class="QWidget" name="InformationDockContents">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>&lt;b&gt;Recent selections&lt;/b&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="historyList">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="clearHistoryButton">
     <property name="text">
      <string>Clear history</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
